server/statistics/hot_cache.go

```diff
@@ -402,16 +407,30 @@ func (w *HotSpotCache) RandHotRegionFromStore(storeID uint64, kind FlowKind, hot
 func (w *HotSpotCache) CollectMetrics(stats *StoresStats) {
 	for storeID, flowStats := range w.writeFlow.hotStoreStats {
 		storeTag := fmt.Sprintf("store-%d", storeID)
-		threshold := calculateWriteHotThresholdWithStore(stats, storeID)
 		hotCacheStatusGauge.WithLabelValues("total_length", storeTag, "write").Set(float64(flowStats.Len()))
-		hotCacheStatusGauge.WithLabelValues("hotThreshold", storeTag, "write").Set(float64(threshold))
+		var topnMin, restMax uint64
+		if stat := flowStats.GetTopNMin(); stat != nil {
+			topnMin = stat.(*HotSpotPeerStat).GetFlowBytes()
+		}
+		if stat := flowStats.GetRestMax(); stat != nil {
+			restMax = stat.(*HotSpotPeerStat).GetFlowBytes()
+		}
+		hotCacheStatusGauge.WithLabelValues("topn_min", storeTag, "write").Set(float64(topnMin))
+		hotCacheStatusGauge.WithLabelValues("rest_max", storeTag, "write").Set(float64(restMax))
```

Comment:
I guess that in most cases the two should be very close, and just recording one is enough.